blas/src/KokkosBlas2_team_spmv.hpp

```diff
@@ -0,0 +1,141 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+#ifndef KOKKOSBLAS2_TEAM_SPMV_HPP_
+#define KOKKOSBLAS2_TEAM_SPMV_HPP_
+
+/// \file KokkosBlas2_spmv.hpp
```

Comment:
This is the source of the failing docs-check

Reply:
Thanks! :)